### ISSUE TITLE
mongorestore and mongodump equivalent functionality

### ISSUE THREAD

**Issue by ipritom** [scottrogowski/mongita#25]
That's an awesome tools, and really very close to MongoDB. Is there any feature equivalent to `mongodump` and `mongorestore` (or `mongoexport` and `mongoimport`)? If not, it would be a useful addition to the module. Thanks.

**Comment by scottrogowski** [comment]
Are you maybe looking for this? https://github.com/scottrogowski/mongita/blob/master/scripts/mongitasync

**Comment by ipritom** [comment]
That's another great functionality I've just come to know from you reply. But I was looking for something to save the entire database in json or bson format. Currently in the `MongitaClientDisk` host location I'm seeing some files with `$` sign in their names. I can dump all the data manually in json/bson file. I was curious if the functionality is already builtin.

**Comment by scottrogowski** [comment]
Nope. Nothing like that is implemented at the moment. I’m curious about your use case though?
